bin/run_analysis.py

```diff
@@ -7,39 +7,33 @@
 import time
 
 from src.run_analysis import analyze
-from src.create_training_data import load_data_from_disk
+from src.create_training_data import load_data_from_disk, CACHE_PATH
 from src.render_results import render_results_for_analysis
-from config_data import CACHE_PATH
 
 def create_parser():
     parser = argparse.ArgumentParser()
     parser.add_argument("--tile-size",
                         default=64,
                         type=int,
                         help="tile the NAIP and training data into NxN tiles with this dimension")
-    parser.add_argument("--training-batches",
-                        default=100,
-                        type=int,
-                        help="the number of batches to train the neural net. 100 is good for a dry "
-                        "run, but around 5000 is recommended for statistical significance")
-    parser.add_argument("--batch-size",
-                        default=100,
-                        type=int,
-                        help="around 100 is a good choice")
     parser.add_argument("--band-list",
                         default=[0, 0, 0, 1],
                         nargs=4,
                         type=int,
                         help="specify which bands to activate (R  G  B  IR). default is "
                         "--bands 0 0 0 1 (which activates only the IR band)")
     parser.add_argument("--render-results",
-                        default=True,
+                        default=False,
                         action='store_true',
```

Comment:
Same thing as above here; you can just call this `--enable-rendering-results` without a default because I don't think the default is doing what you think it's doing.